features/overflow-shorthand.yml

```diff
@@ -25,3 +27,4 @@ compat_features:
   - css.properties.overflow.visible
   - css.types.overflow
   - css.types.overflow.clip
+  - css.types.overflow.overlay
```

Comment:
MDN has this to say about `overlay`:

> Note: The keyword value overlay is a legacy value alias for auto. With overlay, the scroll bars are drawn on top of the content instead of taking up space.

Do you think the value should be marked as deprecated in BCD?

Reply:
Good call. I removed and opened https://github.com/mdn/browser-compat-data/pull/24877